modules/client/src/addresslistVoting/internal/client/methods.ts

```diff
@@ -240,6 +254,99 @@ export class AddresslistVotingClientMethods extends ClientCore
       key: ExecuteProposalStep.DONE,
     };
   }
+  /**
+   * Prepares the installation of a token voting plugin in a given dao
+   *
+   * @param {AddresslistVotingPluginPrepareInstallationParams} params
+   * @return {*}  {AsyncGenerator<PrepareInstallationStepValue>}
+   * @memberof MultisigClientMethods
+   */
+  public async *prepareInstallation(
+    params: AddresslistVotingPluginPrepareInstallationParams,
+  ): AsyncGenerator<PrepareInstallationStepValue> {
+    const signer = this.web3.getConnectedSigner();
+    if (!signer) {
+      throw new NoSignerError();
+    } else if (!signer.provider) {
+      throw new NoProviderError();
+    }
+    const network = await signer.provider.getNetwork();
+    const networkName = network.name as SupportedNetworks;
+    if (!SupportedNetworksArray.includes(networkName)) {
+      throw new UnsupportedNetworkError(networkName);
+    }
+    // connect to psp contract
+    const pspContract = PluginSetupProcessor__factory.connect(
+      LIVE_CONTRACTS[networkName].pluginSetupProcessor,
+      signer,
+    );
+    // connect to plugin repo
+    const addresslistVotingRepoContract = PluginRepo__factory.connect(
+      LIVE_CONTRACTS[networkName].addresslistVotingRepo,
+      signer,
+    );
+    // use specified version or latest
+    let versionTag: VersionTag;
+    if (!params.versionTag) {
+      const latestVersion = await addresslistVotingRepoContract
+        ["getLatestVersion(address)"](
+          LIVE_CONTRACTS[networkName].addresslistVotingSetup,
+        );
+      versionTag = {
+        build: latestVersion.tag.build,
+        release: latestVersion.tag.release,
+      };
+    } else {
+      versionTag = params.versionTag;
+    }
+    // get install data
+    const addresslistVotingPluginInstallItem = AddresslistVotingClientEncoding
+      .getPluginInstallItem(params.settings, networkName);
+    // execute proepareInstallation
+    const tx = await pspContract.prepareInstallation(
+      params.daoAddressOrEns,
+      {
+        pluginSetupRef: {
+          pluginSetupRepo: LIVE_CONTRACTS[networkName].addresslistVotingRepo,
+          versionTag,
+        },
+        data: addresslistVotingPluginInstallItem.data,
+      },
+    );
+
+    yield {
+      key: PrepareInstallationStep.PREPARING,
+      txHash: tx.hash,
+    };
+
+    const receipt = await tx.wait();
+    const pspContractInterface = PluginSetupProcessor__factory
+      .createInterface();
+    const log = findLog(
+      receipt,
+      pspContractInterface,
+      "InstallationPrepared",
+    );
+    if (!log) {
+      throw new ProposalCreationError();
+    }
+
+    const parsedLog = pspContractInterface.parseLog(log);
+    const pluginAddress = parsedLog.args["plugin"];
+    const preparedSetupData = parsedLog.args["preparedSetupData"];
+    if (!(pluginAddress || preparedSetupData)) {
+      throw new PluginInstallationError();
```

Comment:
```suggestion
      throw new PluginInstallationPreparationError();
```

We don't install the plugin here